libs/components/core/src/lib/modules/help/help-update-args.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ *  Options for updating global help.
+ */
+export interface SkyHelpUpdateArgs {
+  /**
+   * A unique key that identifies the current help content to display. If not provided, the page's default help content will be displayed.
+   */
+  helpKey?: string;
+  /**
+   * A unique key that identifies the page's default help content to display.
```

Comment:
I would add something akin to, "Set this property to `undefined` to unset the current page default help key."

Reply:
Done.